.babelrc

```diff
@@ -0,0 +1 @@
+{ "stage": 1 }
```

Comment:
Whoops, you are right.  I was using babel6, I think that babel6 is just backwards-compatible?

I was enabling stage:1 to turn on decorators.